packages/iTwinUI-react/src/core/Select/SelectTag.tsx

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import SvgCloseSmall from '@itwin/itwinui-icons-react/cjs/icons/CloseSmall';
+import { useTheme, CommonProps } from '../utils';
+import '@itwin/itwinui-css/css/inputs.css';
+
+export type SelectTagProps = {
+  /**
+   * Callback function that handles click on close icon.
+   * Close icon is shown only when this function is passed.
+   */
+  onRemove?: React.MouseEventHandler;
+  /**
+   * Text inside the tag.
+   */
+  children: React.ReactNode;
+} & CommonProps;
+
+/**
+ * Tag for showing selected value in `Select`.
+ * @private
+ */
+export const SelectTag = (props: SelectTagProps) => {
+  const { className, children, onRemove, ...rest } = props;
+  useTheme();
+
+  return (
+    <span
+      className={cx('iui-select-tag', className)}
```

Comment:
Why is this truncating even when there is space available?

![ ](https://user-images.githubusercontent.com/9084735/179613825-e9c2a205-aa97-4ca4-a950-bfc68648d670.png)

Reply:
ohh, can you tell me how you reproduced this one??? Test captures this too

Reply:
i can't reproduce it either but i noticed in tests